interface/global-preferences.md

```diff
@@ -0,0 +1,55 @@
+# Global Preferences 
+**Global Preferences** allows us to apply default settings for aspects of our Incari **Project**. These aspects are discussed below:
```

Comment:
"Incari **Project**" - This is misleading because the preferences are global and affect every project, not just the current one.